linter.py

```diff
@@ -21,13 +21,18 @@
 """
 
 from SublimeLinter.lint import Linter
+from os import name
 
 
 class Shellcheck(Linter):
     """Provides an interface to shellcheck."""
 
     syntax = ('shell-unix-generic', 'bash')
     cmd = 'shellcheck --format gcc -'
+
+    if name == "nt":
```

Comment:
FYI we test `sublime.platform() == 'windows'` in Sublime land. Also single quotes.

Reply:
Ah see my last commit in this PR, I changed it to follow your convention. Must have just missed this part. I will make the changes requested.